api/deploy_landing_page.sh

```
@@ -0,0 +1,34 @@
+echo "===================================="
+
+LANDING_DIR="../landing-pages/sudoku-for-seniors"
+
+# Step 1: Ensure lead magnet is in place
+echo "📄 Copying lead magnet..."
+mkdir -p $LANDING_DIR/public/downloads
+cp ../generated/lead_magnets/5_FREE_Sudoku_Puzzles_20250706_114640.pdf $LANDING_DIR/public/downloads/5-free-sudoku-puzzles.pdf
+
+# Step 2: Build the site
+echo "🏗️ Building landing page..."
+cd $LANDING_DIR && npm run build && cd -
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)